pages/management/ManagementHandler.php

```diff
@@ -579,6 +583,25 @@ public function manageEmails(array $args, Request $request): void
         $templateMgr->display('management/manageEmails.tpl');
     }
 
+    /**
+     * Edit or view user using user access table action
+     * @param array $args
+     * @param $request
+     * @return void
+     * @throws \Exception
+     */
+    public function editUser(array $args, $request): void
+    {
+        $this->setupTemplate($request);
+        $invitation = null;
+        if(!empty($args)) {
+            $invitationController = new UserRoleAssignmentInviteRedirectController($invitation);
+            $invitationController->inviteHandle($request,$args[0],'editUser');
```

Comment:
Here, perhaps, define in a separate $variable what the `$args[0]` is, for clarity?

Here, we should also consider standardizing the 'editUser' in a Enumeration instead of the "hardcoded" string?